bitnami/rabbitmq-cluster-operator/templates/messaging-topology-operator/metrics-service.yaml

```diff
@@ -49,5 +49,4 @@ spec:
     {{- end }}
   selector: {{- include "common.labels.matchLabels" . | nindent 4 }}
     app.kubernetes.io/component: messaging-topology-operator
-    app.kubernetes.io/part-of: rabbitmq
```

Comment:
Is there any reason to be deleting this? I won't change this. I think the PR could have the same level of effectiveness just adding the ` app.kubernetes.io/component: messaging-topology-operator` when needed.

Reply:
Consistency with the deployment label selector and overall consistency with chart conventions. Usually you add only the component additionally to the `common.labels.matchLabels` if needed to differ between different components.

Reply:
If it was for me, I would remove the `app.kubernetes.io/part-of: rabbitmq` from the chart completely, as this is something I would only set as a chart user via `commonLabels`, so I can group the resources from the chart how I want :)